dev/com.ibm.ws.security.javaeesec.cdi/resources/com/ibm/ws/security/javaeesec/cdi/internal/resources/JavaEESecMessages.nlsprops

```diff
@@ -30,17 +30,9 @@ JAVAEESEC_CDI_ERROR_UNSUPPORTED_CRED=CWWKS1927E: The supplied credential class {
 JAVAEESEC_CDI_ERROR_UNSUPPORTED_CRED.explanation=The supplied credential class for the authentication is not supported by the user registry. Use either javax.security.enterprise.credential.UsernamePasswordCredential or javax.security.enterprise.credential.BasicAuthenticationCredential for authentication.
 JAVAEESEC_CDI_ERROR_UNSUPPORTED_CRED.useraction=Make sure that the supported credential class is used for authentication by using the user registry.
 
-JAVAEESEC_CDI_ERROR_USERNAME_NULL=CWWKS1928E: Principal name is null or Principal object does not exist. Make sure that CredentialValidationResult contains valid Principal object.
-JAVAEESEC_CDI_ERROR_USERNAME_NULL.explanation=Principal object in the given CredentialValidationResult object is not valid. Either it is null or it does not have valid name.
-JAVAEESEC_CDI_ERROR_USERNAME_NULL.useraction=If the application provides an IdentityStore, make sure that it returns valid Principal object in CredentialValidationResult.
-
-JAVAEESEC_CDI_ERROR_UNIQUE_ID_NULL=CWWKS1929E: UniqueId is null. Make sure that CredentialValidationResult contains valid UniqueId.
-JAVAEESEC_CDI_ERROR_UNIQUE_ID_NULL.explanation=UniqueId in the given CredentialValidationResult object is null.
-JAVAEESEC_CDI_ERROR_UNIQUE_ID_NULL.useraction=If the application provides an IdentityStore, make sure that it returns valid UniqueId in CredentialValidationResult.
-
-JAVAEESEC_CDI_WARNING_NO_IDENTITY_STORE_HANDLER=CWWKS1930W: IdentityStoreHandler bean is not found. Use a User Registry which is defined by server.xml. IdentityStoreHandler bean is created when there is at least one IdentityStore bean. If the application intends to use an IdentityStore, make sure that it is configured properly. 
-JAVAEESEC_CDI_WARNING_NO_IDENTITY_STORE_HANDLER.explanation=IdentityStoreHandler bean is not found. If the application intends to use an IdentityStore, make sure that it is configured properly. IdentityStoreHandler bean is created when there is at least one IdentityStore bean.
-JAVAEESEC_CDI_WARNING_NO_IDENTITY_STORE_HANDLER.useraction=If the application intends to use an IdentityStore, make sure that it is configured properly.
+JAVAEESEC_CDI_WARNING_NO_IDENTITY_STORE=CWWKS1930W: The IdentityStore object configured was not found. A user registry will be used if it is configured. Make sure that the IdentityStore object configured by the application is available.
+JAVAEESEC_CDI_WARNING_NO_IDENTITY_STORE.explanation=The reason that the IdentityStore object was not found varies. The error messages for the IdentityStore object provide information on why the the IdentityStore object was not found.
+JAVAEESEC_CDI_WARNING_NO_IDENTITY_STORE.useraction=Investigate any error messages from the IdentityStore object and make corrections based on the error messages.
```

Comment:
ID review:

JAVAEESEC_CDI_WARNING_NO_IDENTITY_STORE=CWWKS1930W: The configured IdentityStore object was not found. If a user registry is configured, it will be used instead. Make sure that the IdentityStore object configured by the application is available.
JAVAEESEC_CDI_WARNING_NO_IDENTITY_STORE.explanation=The reason that the IdentityStore object was not found varies. The error messages for the IdentityStore object provide information about why the IdentityStore object was not found.
JAVAEESEC_CDI_WARNING_NO_IDENTITY_STORE.useraction=Investigate any error messages from the IdentityStore object, and make corrections based on the error messages.

Reply:
Thank you for your review. I'll update them.